checker/build.gradle

```diff
@@ -221,8 +221,8 @@ artifacts {
 }
 
 clean {
-    delete "${projectDir}/dist"
-    delete 'tests/calledmethods-delomboked'
+    delete('${projectDir}/dist')
```

Comment:
Making this uniform is good, it just needs to continue using double quotes, otherwise the string interpolation won't work.

Reply:
Thanks, I have updated it!